### PR TITLE
[DOCS-94] Update per new credit definitions

### PR DESCRIPTION
**Pull request by mjang-cobalt**
Do not merge until at least late May 31

@yogipetkar This should change everything in our product docs.
Unknown: are we still planning to provide a "Simple" report for 4-7 credits, cc @TheInstaGrahame 

You'll see the proposed changes in the build:

- https://deploy-preview-130--cobalt-docs.netlify.app/getting-started/assets/coverage/
- Each of the (6) methodologies pages below https://deploy-preview-130--cobalt-docs.netlify.app/getting-started/pentest-objectives/methodologies/ 
- https://deploy-preview-130--cobalt-docs.netlify.app/getting-started/checklist/
- https://deploy-preview-130--cobalt-docs.netlify.app/getting-started/glossary/#pentest-report

### CODE REVIEW

File: layouts/shortcodes/min-credits-report.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-We'll write a report for pentests with at least two (2) credits
-<!-- Change for the new credit implementation on May 31 -->
\ No newline at end of file
+We'll write a report for pentests with at least four (4) credits
```
Comment (mjang-cobalt) [inline layouts/shortcodes/min-credits-report.md:1]: As suggested by @TheInstaGrahame 

```suggestion
We'll write a report for pentests with at least five (5) credits
```

File: layouts/shortcodes/pentest-report-requirements.html [hunk 2]
```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least two (2) credits. 
-If you have a one (1) credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least four (4) credits.
```
Comment (mjang-cobalt) [inline layouts/shortcodes/pentest-report-requirements.html:1]: As suggested by @TheInstaGrahame 

```suggestion
If you want a pentest report, you generally must set up a test of at least five (5) credits. 
```

Comment (yogipetkar) [reply layouts/shortcodes/pentest-report-requirements.html:1]: > As suggested by @TheInstaGrahame

New Credit definition in platform does not support incremental credits such as 5,6,7,8,.... Such support will be available for assurance release. Until then for NCD release the credit increments are 4,8,12,16,... increments of 4. If PenOps plans to provide a report for a 4 credit pentest by additionally deducting 1 credit through Admin interface then this makes sense.

File: layouts/shortcodes/min-credits-report.md (hunk 1 above)
Comment (mjang-cobalt) [reply layouts/shortcodes/min-credits-report.md:1]: After discussion with @TheInstaGrahame and @yogipetkar 

```suggestion
We'll write a report for pentests with at least eight (8) credits
```

File: layouts/shortcodes/pentest-report-requirements.html (hunk 2 above)
Comment (mjang-cobalt) [reply layouts/shortcodes/pentest-report-requirements.html:1]: After discussion with @TheInstaGrahame and @yogipetkar 

```suggestion
If you want a pentest report, you generally must set up a test of at least eight (8) credits. 
```

File: layouts/shortcodes/pentest-report-requirements.html [hunk 3]
```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least two (2) credits. 
-If you have a one (1) credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least five (5) credits.
```
Comment (mjang-cobalt) [inline layouts/shortcodes/pentest-report-requirements.html:1]:
```suggestion
If you want a pentest report, you generally must set up a test of at least eight (8) credits. 
```

File: layouts/shortcodes/pentest-report-requirements.html [hunk 4]
```diff
@@ -1,3 +1,3 @@
-If you want a pentest report, you generally must set up a test of at least two (2) credits. 
-If you have a one (1) credit pentest, you'll still have access to the non-report
+If you want a pentest report, you generally must set up a test of at least eight (8) credits.
```
Comment (mjang-cobalt) [inline layouts/shortcodes/pentest-report-requirements.html:1]: Probably @yogipetkar or @TheInstaGrahame highlighted an error here in some internal chat. I think I've fixed this now, and await your approval.

**Pull request merged**